Example/YelpAPI.xcodeproj/project.pbxproj

```diff
@@ -124,6 +128,8 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				0B5080851CBEEA950071508E /* OHHTTPStubs.framework in Frameworks */,
+				0B5080831CBEEA800071508E /* OCMock.framework in Frameworks */,
```

Comment:
But you should revert commit f296880c756f733b89ecb1fe9eb4eea55989baaa before we merge this!